project-tracker backend/src/app/controllers/user.ts

```diff
@@ -20,7 +22,7 @@ export const index = async (req: Request, res: Response) => {
 
 export const show = async (req: Request, res: Response) => {
     try {
-        const user: User = await database(TableNames.users).select().where({id: req.params.id}).first();
+        const user: User = await database(TableNames.users).select().where({id: req.params.id, deletedAt: null}).first();
         if (user) {
             res.json(userSerializer.show(user));
```

Comment:
.status kimaradt!

Reply:
ha a res.sendStatus-ra gondolsz, nem maradt ki. Órán sem küldtünk.
https://github.com/rastasi/theta-express/blob/master/src/app/controllers/user.ts

Reply:
nem arra gondolok, hanem arra, hogy a json-nek mondjuk meg a biztonság kedvéért mindig, hogy milyen statusszal megy ki. Akkor is, ha a default az 200 és ide pont az kell. A sendStatus-ra nem is gondolhattam volna, mivel az vissza is küldi a választ és nem tudsz utána json-t küldeni. status.

Reply:
Megnéztem a linket, valóban nincs ezekben bent :(
Pedig valahogy így a tuti: res.status(200).json(obj)

Reply:
javítja. commit 9aeb433